### PR TITLE
Feat/python

### PR DESCRIPTION
**Pull request by BotellaA**

### CODE REVIEW

File: bindings/python/src/geosciences/geosciences.cpp [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
```
Comment (panquez) [inline bindings/python/src/geosciences/geosciences.cpp:25]: what is this for?

File: bindings/python/tests/geosciences/test-py-cross-section.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        raise ValueError("[Test] Number of fault blocks in CrossSection should be 2")
+
+def add_stratigraphic_units( model, builder ):
+    stratigraphic_unit0 = builder.add_stratigraphic_unit()
+    stratigraphic_unit1 = builder.add_stratigraphic_unit()
+    builder.add_stratigraphic_unit()
+    builder.set_stratigraphic_unit_name( stratigraphic_unit1, "cretaceous")
+    if model.nb_stratigraphic_units() != 3:
+        raise ValueError("[Test] Number of stratigraphic_units in CrossSection should be 3")
+    builder.remove_stratigraphic_unit(     model.stratigraphic_unit( stratigraphic_unit0 ) )
```
Comment (panquez) [inline bindings/python/tests/geosciences/test-py-cross-section.py:75]:
```suggestion
    builder.remove_stratigraphic_unit( model.stratigraphic_unit( stratigraphic_unit0 ) )
```

File: bindings/python/src/geosciences/geosciences.cpp (hunk 1 above)
Comment (BotellaA) [reply bindings/python/src/geosciences/geosciences.cpp:25]: to handle stl type such as vector. It is used for methods, such as` model.items`, where I return a vector of Components.

**Pull request merged**